### PR TITLE
Created parents registration component and event route.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@ swift:
 
 eslint:
   enabled: true
-  config_file: .eslintrc
+  config_file: .eslintrc.js
```
Comment (pepcus-pratik) [inline .hound.yml]: Is this change required??

File: src/components/ParentsRegistrationForm.jsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        <div>
+          {'All fields are compulsory'}
+        </div>);
+    }
+    return null;
+  }
+  renderPopUp() {
+    if (this.state.isSubmitTriggered) {
+      return (
+        <div div className="inputFieldContainer">
```
Comment (pepcus-pratik) [inline src/components/ParentsRegistrationForm.jsx]:
```suggestion
        <div className="inputFieldContainer">
```

File: .hound.yml (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply .hound.yml]: no

File: src/components/ParentsRegistrationForm.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/ParentsRegistrationForm.jsx]: Done

File: src/sagas/rootSaga.js [hunk 3]
```diff
@@ -194,14 +198,29 @@ export function* markSelectedStudentsOptInOrOptOutSaga(action) {
...
     yield put(updateIdCardStatusSelectedStudentsResultsFailureAction(errorMessage));
   }
 }
+
+export function* parentsRegistrationSaga(action) {
+  const { name, members, phoneNumber } = action;
+  const errorMessage = 'Error getting registration.';
+  try {
+    const response = yield parentsRegistrationAPI(name, members, phoneNumber);
+    if (response.message === 'Registration successful') {
```
Comment (pepcus-pratik) [inline src/sagas/rootSaga.js]: Check should not be on the **message**
